src/test/java/runner/utilTestClasses/TestRunner.java

```diff
@@ -1,12 +1,12 @@
 package runner.utilTestClasses;
 
-import edu.uw.cse.testbayes.runner.IndividualClassRunner;
+import edu.uw.cse.testbayes.runner.TestBayesIndividualClassRunner;
 import org.junit.Ignore;
 import org.junit.runner.RunWith;
-import org.junit.runners.*;
+import org.junit.runners.Suite;
 
 @Ignore
-@RunWith(Suite.class)
+@RunWith(TestBayesIndividualClassRunner.class)
```

Comment:
@emayer2 Need you to confirm what class comes into the `@RunWith` annotation

Reply:
I'm doing to change the readme, since our main class is what's going to run suite classes